sample-lnd.conf

```diff
@@ -1472,6 +1472,9 @@
 ; own risk.
 ; db.use-native-sql=false
 
+; If set to true, native SQL invoice migration will be skipped. Note that this
+; option is intended for users who experience non-resolvable migration errors.
+; db.skip-sql-invoice-migration=false
```

Comment:
so i guess the big question here is: if a user sets this once... then they can never really unset it/try again yeah?  since the migration version will have moved on. So this is really worst case worst case? i'd hope that we can eventually remove this option...

Reply:
ah - i see you covered this in  a comment above :)

Reply:
is our strategy to right specific fix code to unblock anyone who runs into issues so that they can eventually get unstuck?

Reply:
> so i guess the big question here is: if a user sets this once... then they can never really unset it/try again yeah? since the migration version will have moved on. So this is really worst case worst case? i'd hope that we can eventually remove this option...

Exactly. This is more like a kill-switch in case someone's node gets stuck so they can still start up.

> is our strategy to right specific fix code to unblock anyone who runs into issues so that they can eventually get unstuck?

I really do hope we won't need to but ultimately yes :)